openlibrary/templates/merge_request_table/table_row.html

```diff
@@ -0,0 +1,79 @@
+
+        $# Start : Most recent comment preview
+        <div class="mr-details__comment-preview">
+          $ latest_comment = comments[-1]['message'] if comments else _('No comments yet.')
+          $latest_comment
+        </div>
+        $# End : Most recent comment preview
+
+        $# Start : Comment section
+        $# XXX : Replace this with a modal form
+        <div class="comment-panel hidden">
+          <div class="comment-panel__comment-display">
+            $for c in comments:
+              <div class="comment-panel__comment"><span class="commenter">@$c['username']</span> $c['message']</div>
+          </div>
+          $if is_submitter or can_merge:
+            <div class="comment-panel__input">
+              $# XXX: Ensure that this has a drag handle
+              <textarea class="comment-panel__reply-input" rows="1" placeholder="$_('Add a comment...')"></textarea>
+              <input class="comment-panel__reply-btn" type="button" value="$_('Reply')">
+            </div>
+        </div>
+        $# End : Comment section
+
+        $# Start : Submitted by line
+        <span class="mr-details__opened-by">
+          $:_('MR #%(id)s opened %(date)s by <a href="/people/%(submitter)s" class="commenter">@%(submitter)s</a>', id=request['id'], date=datestr(request["created"]), submitter=request["submitter"])
+          $if is_open and is_submitter:
+            <a class="mr-close-link" title="$_('Click to close this Merge Request')" href="javascript:;">&times;</a>
+        </span>
+        $# End : Submitted by line
+      </div>
+    </td>
+    <td class="mr-review-actions">
+      $ show_review_button = is_open and can_merge and not request.get('reviewer', '')
+      <div class="mr-review-actions__assignee $('hidden' if show_review_button else '')">
+        $ request_reviewer = request.get('reviewer', '')
+        $if request_reviewer:
+          $ request_reviewer = '@' + request_reviewer
+        $if is_open:
+          $ icon_classes = 'mr-review-actions__unassign'
+          $if not can_merge:
+            $ icon_classes = '%s hidden' % icon_classes
+          $ icon_content = '&times;'
+        $else:
+          $ icon_classes = 'mr-review-actions__checkmark'
+          $ icon_content = '✓'
+
+        <span class="mr-review-actions__assignee-name">$request_reviewer</span>
+        <span class="$icon_classes">$:icon_content</span>
+      </div>
+      <a href="$review_url" target="_blank" class="cta-btn mr-review-actions__review-btn $('' if show_review_button else 'hidden')">$_('Review')</a>
+    </td>
+    <td class="mr-comment-toggle">
+      <a href="javascript:;" class="mr-comment-toggle__comment-expand cta-btn">
+        <span class="mr-comment-toggle__comment-count">$len(comments)</span>
+        <img class="mr-comment-review-cell__speech mr-comment-toggle__speech-bubble" src="/static/images/speech-bubble.svg" alt="">
+      </a>
+    </td>
+  </tr>
+  
```

Comment:
```suggestion

```
The standard is that the last line of a file should contain **only** a newline character.  Remove the spaces from line 79, leaving only the newline character.